packages/actor-query-operation-group/examples/README.md

```diff
@@ -0,0 +1,5 @@
+# Group examples
```

Comment:
I would not add these examples to the repo. I think it would make more sense to add these somewhere in the sparqlalgebrajs repo, for example on a wiki page.